sonic3k.asm

```diff
@@ -8965,7 +8965,7 @@ LoadSolids:
 	if Sonic3_Complete=0
 		cmpi.l	#$260000,d0
```

Comment:
Maybe this one should also have its own label?

Reply:
It seems that for some reason, they very deliberately used an address within the lock-on ROM space rather than just point at the boundary. But what would you call that arbitrary address, other than LockOnROM_Start+$60000? I don't have my disasm on me right now, is that where the solid data begins within the S3 ROM?

Reply:
Thinking about it a bit, I think that a better way to handle this in Sonic3_Complete mode (instead of how it is currently done) would be to add a flag on SolidIndexes (maybe the sign bit) and check for it instead of relying on position. This would allow moving data around in ROM without breaking this code path.

And thinking more, most uses of LockOnROM_Start should be replaced by constants defined in "LockOn Data.asm" and merged with their non-Sonic3_Complete versions.

It is a short enough change that I will make tonight, when I am back home.